bot/jobs.py

```diff
@@ -287,8 +291,8 @@ def check_new_goverance_proposal(context):
     for i in range(new_proposals_count):
         current_proposal = governance_proposals[user_data['governance_proposals_count'] + i]
 
-        voting_start_time = datetime.strptime(current_proposal['voting_start_time'][:-4], "%Y-%m-%dT%H:%M:%S.%f")
-        voting_end_time = datetime.strptime(current_proposal['voting_end_time'][:-4], "%Y-%m-%dT%H:%M:%S.%f")
+        voting_start_time = terra_timestamp_to_datetime(current_proposal['voting_start_time'])
```

Comment:
Here I also get the uncaught error
`ValueError: time data '0001-01-01T00:00' does not match format '%Y-%m-%dT%H:%M:%S.%f'`